### PR TITLE
try to use bcc lib on ubuntu in unit test

### PR DESCRIPTION
**Pull request by SamYuan1990**
try to use bcc lib on ubuntu in unit test

Signed-off-by: Sam Yuan <yy19902439@126.com>

### CODE REVIEW

File: .github/workflows/unit_test.yml [hunk 1]
```diff
@@ -65,22 +65,7 @@ jobs:
...
-          # Symlink /usr/lib/llvm to avoid "Unable to find clang libraries"
-          # The directory appears only to be created when installing the
-          # virtual llvm-dev package.
-          # https://github.com/iovisor/bcc/issues/492
-          sudo ln -s /usr/lib/llvm-9 /usr/local/llvm
-          cmake ..
-          make
-          sudo make install
-          popd
+          sudo apt-get install -y libbpfcc-dev linux-headers-$(uname -r)
```
Comment (rootfs) [inline .github/workflows/unit_test.yml:68]: libbpfcc-dev is not updated, it stays at 0.12 :(

Comment (SamYuan1990) [reply .github/workflows/unit_test.yml:68]: yea... I found it.

**Comment by marceloamaral** [comment]
@SamYuan1990, could you try following the Kubernetes commit message guideline?
https://www.kubernetes.dev/docs/guide/pull-requests/#commit-message-guidelines

It's good to provide meaningful messages with context!

**Comment by SamYuan1990** [comment]
> https://www.kubernetes.dev/docs/guide/pull-requests/#commit-message-guidelines

@marceloamaral , would like to config commit template and pr template for us?

**Comment by marceloamaral** [comment]
We don't need a template, just follow the guideline will be enough :)

**Pull request closed without merging**